config/ext_jenkins-weekly.yaml

```diff
@@ -0,0 +1,176 @@
+
+  JCasC:
+    enabled: true
+    defaultConfig: false
+    configScripts:
+      security: |
+        security:
+          scriptApproval:
+            approvedSignatures:
+              - "method org.jenkinsci.plugins.workflow.steps.FlowInterruptedException getCauses"
+              - "new java.lang.RuntimeException java.lang.Throwable"
+      credentials: |
+        credentials:
+          system:
+            domainCredentials:
+              - credentials:
+                - gitHubApp:
+                    appID: "${GITHUB_APP_ID}"
+                    description: "GitHub App for weekly.ci.jenkins.io"
+                    id: "github-app-weekly"
+                    privateKey: "${GITHUB_APP_PRIVATE_KEY}"
+                    scope: GLOBAL
+      agent-settings: |
+        jenkins:
+          numExecutors: 1 # Allow one build on controller for now
+          clouds: []
+      ldap-settings: |
+        jenkins:
+          securityRealm:
+            ldap:
+              configurations:
+                - server: "${LDAP_SERVER}"
+                  rootDN: "${LDAP_ROOT_DN}"
+                  managerDN: "${LDAP_MANAGER_DN}"
+                  managerPasswordSecret: "${LDAP_MANAGER_PASSWORD}"
+                  mailAddressAttributeName: "mail"
+                  userSearch: cn={0}
+                  userSearchBase: "ou=people"
+                  groupSearchBase: "ou=groups"
+              disableMailAddressResolver: false
+              groupIdStrategy: "caseInsensitive"
+              userIdStrategy: "caseInsensitive"
+              cache:
+                size: 100
+                ttl: 300
+      advisor-settings: |
+        jenkins:
+          disabledAdministrativeMonitors:
+            - com.cloudbees.jenkins.plugins.advisor.Reminder
+        advisor:
+          acceptToS: true
+          ccs:
+          - "damien.duportal@gmail.com"
+          email: "jenkins-infra-team@googlegroups.com"
+          excludedComponents:
+            - "ItemsContent"
+            - "GCLogs"
+            - "Agents"
+            - "RootCAs"
+            - "SlaveLogs"
+            - "HeapUsageHistogram"
+          nagDisabled: true
+      pipeline-library: |
+        unclassified:
+          location:
+            url: "https://weekly.ci.jenkins.io"
+          globalLibraries:
+            libraries:
+            - defaultVersion: "master"
+              implicit: true
+              name: "pipeline-library"
+              retriever:
+                modernSCM:
+                  scm:
+                    git:
+                      id: "github-app-weekly"
+                      remote: "https://github.com/jenkins-infra/pipeline-library.git"
+      matrix-settings: |
+        jenkins:
+          authorizationStrategy:
+            globalMatrix:
+              permissions:
+                - "GROUP:Job/Read:anonymous"
+                - "GROUP:Overall/Read:anonymous"
```

Comment:
Unless I'm mistaken, we should also add the administration rights (otherwise no one would be able to operate the instance. Might not be a bad idea though :trollface: )

Reply:
Yes please that needs re-adding it was there before